website/blog/2023-01-17-ferretdb-0-8-1-version-release.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "FerretDB new version release - 0.8.1"
+slug: ferretdb-0-8-1-version-release
+author: Alexander Fashakin
+description: "FerretDB 0.8.1 - the open-source MongoDB alternative - includes new features like version availability, `distinct` command & client TLS validation, and much more."
+image: ../static/img/blog/FerretDB_v0.8.1.jpg
+tags: [release]
+---
+
+![FerretDB version 0.8.1](../static/img/blog/FerretDB_v0.8.1.jpg)
+
+<!--truncate-->
+
+[FerretDB](https://www.ferretdb.io/) 0.8.1 is here, and it's better than ever!
+On the back of our Beta release, we've gone even further to improve FerretDB.
+And why shouldn’t we, especially when you consider the current economical landscape and the need for truly open source software?
```

Comment:
I would remove this sentence. It is great that we are making the case for FerretDB, but probably this is too far into an opinion blog post vs. a release blog post.